guides/doc-Release_notes/topics/katello.adoc

```diff
@@ -4,6 +4,20 @@
 [id="katello-headline-features"]
 == Headline Features
 
+For the full list of changes, see the https://github.com/Katello/katello/blob/KATELLO-{KatelloVersion}/CHANGELOG.md[Changelog].
+
+* Content view versions may now be removed from a content view in bulk via the UI.
+* A new SmartProxy Content download-policy `Streamed` has been added. This will act like a forwarding-proxy. All packages requested will be requested from the main server. This is useful if the SmartProxy does not have enough disk-space to hold all package-contents.
```

Comment:
```suggestion
* A new Smart Proxy Content download-policy `Streamed` has been added. This will act like a forwarding-proxy. All packages requested will be requested from the main server. This is useful if the SmartProxy does not have enough disk-space to hold all package-contents.
```